packages/cli/src/util/loader.ts

```diff
@@ -0,0 +1,9 @@
+import { CannonRegistry, IPFSLoader as CannonIPFSLoader } from '@usecannon/builder';
+
+export class IPFSLoader extends CannonIPFSLoader {
+  constructor(ipfsUrl: string, resolver: CannonRegistry) {
```

Comment:
kind of a wierd way to do it, would rather just have a function like `getIpfsLoader()`